app/javascript/pages/components/Calendar.jsx

```diff
@@ -207,9 +230,15 @@ class Calendar extends React.Component {
           updateTicker(allKeyframes[i], transition);
           await transition.end();
         }
+        d3.select('.replay__container')
+          .attr('visibility', 'visible');
       }
 
-      codeToRun();
+      runChart().then(() => {
+        d3.select('.replay__container')
+          .attr('visibility', 'visible');
+        document.querySelector('.replay__container').addEventListener('click', runChart);
```

Comment:
Nit: You should be able to refactor this by adding `.on()` to the d3.select if you want to make this more concise.

https://www.tutorialsteacher.com/d3js/event-handling-in-d3js

Reply:
I tried this yesterday evening and for some reason kept struggling with it; going to keep this for now (for the sake of quick turnaround) but duly noted.